src/meroxa/resources.py

```diff
@@ -24,7 +28,7 @@ def __init__(
         url: str,
         metadata: dict,
         connector_count: int,
-        status: Status,
+        status: Any,
```

Comment:
shouldn't this still be Status?
```suggestion
        status: Status,
```

Reply:
This would not be an object by the time the response object gets instantiated.  

API Response -> JSON Blob -> Unmarshaled into response object -> nested dictionary gets unmarshaled into `Status` object. 

Typing this out now, I'm realizing that we might not even need an object but it helps to have the expected fields explicitly defined.